Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsFragment.java

```diff
@@ -9,24 +9,20 @@
 import android.os.Looper;
 import android.preference.PreferenceManager;
 import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
```

Comment:
not needed

Reply:
I guess it is used becouse of Snackbar.LENGTH_LONG